Tests/XMLCoderTests/CombineTests.swift

```diff
@@ -27,6 +27,12 @@ private struct Foo: Codable {
     var name: String
 }
 
+final class CustomEncoder: XMLEncoder {
+    override func encode<T>(_ value: T) throws -> Data where T : Encodable {
+        try self.encode(value, withRootKey: "bar", rootAttributes: nil, header: nil)
```

Comment:
```suggestion
        return try self.encode(value, withRootKey: "bar", rootAttributes: nil, header: nil)
```

Reply:
I fixed in 84c33e77a57cd008c86321476f5e5a506d80fd76.